Mods/ModVarietyPack/Defs/TraderKindDefs/TraderKinds.xml

```diff
@@ -0,0 +1,1303 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
would be cool if we used CCL method instead. See how CombatRealism did it in its ModHelper.xml